laura/portfolio/src/main/webapp/mood.html

```diff
@@ -6,6 +6,12 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
+
+    <ul>
+        <li><a href="index.html" class="navbar">Home</a>
+            <a href="mood.html" class="navbar">Mood generator</a></li>
+    </ul>
+
```

Comment:
Same as for the main page, navbar should be inside <body> and you can have look at "<nav>" tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav